src/features/counter/counterAPI.js

```diff
@@ -1,6 +1,48 @@
-// A mock function to mimic making an async request for data
-export function fetchCount(amount = 1) {
+const url = "https://qc-engine.herokuapp.com/api";
+
+const checkResponse = (response) => {
+  if (!response.ok) {
+    throw new Error('The personnel isn\'t currently available.');
+  } else {
+    return response.json();
+  }
+}
+const contactEndpoint = "/v1/contacts/"
+
+
+const getContactById = (id) => {
+  return {
+    "id": "1",
+    "type": "contact",
+    "attributes": {
+      "name": "Moo Philips",
+      "email": "mphillips@email.gov",
+      "phone_number": "123-456-7890",
+      "job_title": "Incident Commander",
+    }
+  }
+}
+
+const getContactByName = (name) => {
+  // fetch(`${url}/v1/contacts?name=${name}`)
+  //  .then(checkResponse)
   return new Promise((resolve) =>
```

Comment:
Good to have a sense of how these network requests will look once the backend's up and running